.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,3 +14,6 @@ Fixes:
 - [ ] Meaningful title, helpful description and a linked NuGet/Home issue
 - [ ] Added tests
 - [ ] Link to an issue or pull request to update docs if this PR changes settings, environment variables, new feature, etc.
+- [ ] If the PR adds UI:
```

Comment:
Our PR template has evolved over the years, but recent learnings/adjustments are that people won't fill it out if the template is too long. 

In https://github.com/NuGet/NuGet.Client/pull/5907, we simplified it to the 3 checkboxes that we have today. 
The idea with those 3 is that they're the most common and apply to a large majority of PRs. 

UI PRs aren't the majority of changes, so I don't think it's something that should be part of the template that covers all kinds of PRs. 

That being said, we do have places where call out localization and accessibility passes in our developer docs. IN particular, https://github.com/NuGet/NuGet.Client/blob/dev/docs/feature-guide.md#visual-studio-ui-considerations. 
We also have https://github.com/NuGet/NuGet.Client/blob/dev/docs/ui-guidelines.md which goes in a lot more detail. Maybe we can improve the reachability of this documentation?